### PR TITLE
[AMORO-2910] Add profile to skip WebUI build process

### LINKED ISSUE #2910: [Improvement]: Add profile to skip WebUI build process
### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/amoro/issues?q=is%3Aissue) and found no similar issues.

### What would you like to be improved?

When debug some issues, we need to build the project many times for debugging, so add this profile to speed up the build.

### How should we improve?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Subtasks

_No response_

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by xleoken**
## Why are the changes needed?

Close #2910.

## Brief change log

We need to build the project many times for debug some issues, so add this profile to speed up the build.

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: amoro-ams/amoro-ams-dashboard/pom.xml [hunk 1]
```diff
@@ -113,4 +113,22 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>skip-webui-build</id>
```
Comment (zhoujinsong) [inline amoro-ams/amoro-ams-dashboard/pom.xml:118]: Maybe `skip-ams-dashboard-build` is a proper name as the module name is `amoro-ams-dashboard`.

**Review by zhoujinsong — COMMENTED** [review]
@xleoken Thanks for the contribution!

I left a suggestion, PTAL.

File: amoro-ams/amoro-ams-dashboard/pom.xml [hunk 2]
```diff
@@ -113,4 +113,22 @@
...
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>com.github.eirslett</groupId>
+                            <artifactId>frontend-maven-plugin</artifactId>
+                            <configuration>
+                                <skip>true</skip>
```
Comment (czy006) [inline amoro-ams/amoro-ams-dashboard/pom.xml:126]: I think this configuration should be turned off, **false** by default.

File: amoro-ams/amoro-ams-dashboard/pom.xml (hunk 1 above)
Comment (xleoken) [reply amoro-ams/amoro-ams-dashboard/pom.xml:118]: hi @zhoujinsong, `skip-ams-dashboard-build` seems too long for users. 

https://github.com/apache/flink/blob/b72ce532fdd77bca86a030be1b9878be4c097546/flink-runtime-web/pom.xml#L113

Comment (zhoujinsong) [reply amoro-ams/amoro-ams-dashboard/pom.xml:118]: Then how about `skip-dashbord-build`?

File: amoro-ams/amoro-ams-dashboard/pom.xml (hunk 2 above)
Comment (xleoken) [reply amoro-ams/amoro-ams-dashboard/pom.xml:126]: hi @czy006, this profile will not affect the default value.

https://github.com/apache/flink/blob/b72ce532fdd77bca86a030be1b9878be4c097546/flink-runtime-web/pom.xml#L113

File: amoro-ams/amoro-ams-dashboard/pom.xml (hunk 1 above)
Comment (xleoken) [reply amoro-ams/amoro-ams-dashboard/pom.xml]: > Then how about `skip-dashbord-build`?

Make sense, updated to `skip-dashboard-build`.

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Pull request merged**